src/components/notifications/WebhookConfigModal.tsx

```diff
@@ -153,7 +151,7 @@ export class WebhookConfigModal extends Component<WebhookConfigModalProps, Webhh
         }
         let requestBody = this.state.form;
         if (this.props.webhookConfigId) requestBody['id'] = this.props.webhookConfigId;
-        let promise = this.props.webhookConfigId ? updateWebhookConfiguration(requestBody) : saveWebhookConfiguration(requestBody);
+        let promise = this.props.webhookConfigId ? saveUpdateWebhookConfiguration(requestBody, true) : saveUpdateWebhookConfiguration(requestBody, false);
```

Comment:
we can directly write `const promise = saveUpdateWebhookConfiguration(requestBody, !!this.props.webhookConfigId)`. Even we can directly use this without creating variable

Reply:
done